datadog_checks_dev/datadog_checks/dev/tooling/commands/release.py

```diff
@@ -669,7 +677,7 @@ def make(ctx, checks, version, initial_release, skip_sign, sign_only):
         echo_success('success!')
 
         commit_targets = [check]
-
+        updated_checks.append(check)
```

Comment:
Not sure, but should `updated_checks.append(check)` be placed inside `if check not in NOT_CHECKS` ?

Reply:
Good question, this condition is only for `datadog_checks_dev` that is not packaged with the agent. Looks like today we do sign and upload those wheels though. But is that required? cc @ofek

Reply:
Not required but let's keep doing it

Reply:
Then we're all good here 👍